pootle/apps/pootle_store/unit/timeline.py

```diff
@@ -257,4 +295,7 @@ def get_grouped_entries(self):
         return grouped_entries
 
     def get_entry(self, item):
-        return self.entry_class(ProxySubmission(item)).entry
+        submission = ProxySubmission(item)
```

Comment:
im thinking just include it in the item dict

```
item["comment"] = self.get_comment_for_item(item)
```
and then add a `suggestion_comment` method to `ProxySubmission`